cinema/views.py

```diff
@@ -1,3 +1,5 @@
+from wsgiref.util import request_uri
```

Comment:
What is this and why you've imported this?

Reply:
Sorry, it probably got imported by itself because of pycharm's autocomplete